stable/database/templates/deployment.yaml

```diff
@@ -37,6 +37,10 @@ spec:
       priorityClassName: {{ default "" .Values.database.priorityClasses.te }}
       {{- end }}
       {{- include "securityContext" . | indent 6 }}
+      {{- if .Values.database.te.topologySpreadConstraints }}
+      topologySpreadConstraints:
+{{ tpl .Values.database.te.topologySpreadConstraints . | trim | indent 8 }}
```

Comment:
You could preserve the indentation with:
`{{- tpl .Values.database.te.topologySpreadConstraints . | trim | nindent 8 }}`

Reply:
indent is used in other areas for similar sections such as affinity and tolerations.